maplibre_helpers.ts

```diff
@@ -31,7 +32,11 @@ export function drawPolygon(color, opacity) {
   };
 }
 
-export function drawCircle(color, radius, opacity = 1.0) {
+export function drawCircle(
+  color,
+  radius,
+  opacity: DataDrivenPropertyValueSpecification<number> = 1.0
```

Comment:
Because we call this method a few times with one of those long `["case", ...]` expressions, we need to do this. Figuring out the name of this type was not easy; I couldn't find the TS types anywhere online in the normal docs. So I just grepped the maplibregl code (conveniently in `node_modules` for `circle-opacity` and figured this out